source/rust_verify/src/config.rs

```diff
@@ -150,8 +151,10 @@ pub fn parse_args(program: &String, args: impl Iterator<Item = String>) -> (Args
     const OPT_PROFILE_ALL: &str = "profile-all";
     const OPT_COMPILE: &str = "compile";
     const OPT_NO_SOLVER_VERSION_CHECK: &str = "no-solver-version-check";
+    const OPT_VERSION: &str = "version";
 
     let mut opts = Options::new();
+    opts.optflag("", OPT_VERSION, "print version information");
```

Comment:
For consistency with the capitalization of other options
```suggestion
    opts.optflag("", OPT_VERSION, "Print version information");
```